### PR TITLE
Completed test coverage for django.utils.encoding.

### PR DESCRIPTION
**Pull request by timgraham**

### CODE REVIEW

File: tests/utils_tests/test_encoding.py [hunk 1]
```diff
@@ -49,20 +50,30 @@ def test_force_bytes_strings_only(self):
...
+        error_msg = 'This is an exception, voilà'.encode('utf-8')
+        result = force_bytes(error_msg, encoding='ascii', errors='ignore')
+        self.assertEqual(result, b'This is an exception, voil')
+
+    def test_smart_bytes(self):
         class Test:
             def __str__(self):
                 return 'ŠĐĆŽćžšđ'
 
-        class TestU:
```
Comment (timgraham) [inline tests/utils_tests/test_encoding.py]: I'm not positive whether or not `TestU` is obsolete but there were more differences between `Test` and `TestU` when Python 2 was supported.

**Comment by timgraham** [comment]
@claudep, any comments?

**Comment by claudep** [comment]
Huh... `smart_bytes` is going to vanish. It's completely unused by Django itself, so I'd rather not add tests for it now.

**Comment by timgraham** [comment]
I assume it needs to go through a deprecation so it seems nice not to have a hole in the coverage report until it's removed.

**Pull request merged**